Vagrantfile

```diff
@@ -20,5 +20,15 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       vb.gui = false
     end
   end
+  config.vm.define "pwn32", autostart: false do |u32|
+    u32.vm.box = "puphpet/ubuntu1604-i386"
+    u32.vm.network "private_network", ip: "10.10.10.11"
+    u32.vm.provider "virtualbox" do |vb|
+      vb.customize ['modifyvm', :id, '--cableconnected1', 'on']
```

Comment:
I had to add that line in order to ssh into my vm, because by default the vagrant box comes without the cable connected on the vb network interface. Not sure if it is a vagrant issue or vb issue.

I'm also open to trying other 32-bit Xenial vagrant boxes, btw if we can find a better one.